catalog/RUB/RUB_RUBCLIM_LCZ_global_lcz_map_v1.jsonnet

```diff
@@ -0,0 +1,335 @@
+local id = 'RUB/RUBCLIM/LCZ/global_lcz_map/v1';
+local subdir = 'RUB';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'Global map of Local Climate Zones',
+  version: 'v1',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    Since their introduction in 2012, Local Climate Zones (LCZs) emerged as a new standard
```

Comment:
From reading this, I'm not sure I understand how bands LCZ and LCZ_Filter differ. Which one should the majority of people use?

Reply:
LCZ_Filter is a Gaussian filtered version of the LCZ band. This is the recommended product to use. 

I tried to make this clear in the description:
```
The recommended global LCZ map, with LCZ classes indicated by numbers 1-17. LCZ labels are obtained after applying the morphological Gaussian filter described in Demuzere et al. (2020)
```

Is this ok, or should this be done in a different way?

Reply:
I think this is too far in the band descriptions, and many people will miss this text. I would write something incredibly obvious in the main description text, e.g.: "LCZ_Filter is the recommended band fo rmost users. The other classification band, LCZ, is only provided for ..."

(I'd also make LCZ_Filter the first band, but given how much trouble we are having with exports, I won't insist on that.)

Reply:
Ok. I have added this explicit statement in the general description.

If it is fine by you, I will not be re-processing the exports. The code snippet also uses the LCZ_Filter band, so I assume this will be clear now!